src/main/scala/wdltool/Main.scala

```diff
@@ -72,6 +74,76 @@ object Main extends App {
     }
   }
 
+  def graph(args: Seq[String]): Termination = {
```

Comment:
I'd prefer to see the graph logic out of Main (yes, the other stuff is here too but the other stuff isn't as long) but don't care that much if you decide to not bother